rotkehlchen/tests/api/blockchain/test_evm.py

```diff
@@ -290,30 +291,25 @@ def new_modify_blockchain_accounts(blockchain, accounts, append_or_remove) -> Ca
         ), json=request_data)
 
     result = assert_proper_response_with_result(response)
+
     assert result == {
         'added': {
             '0x9531C059098e3d194fF87FebB587aB07B30B1306': ['all'],
             '0x9008D19f58AAbD9eD0D60971565AA8510560ab41': ['eth'],
+            '0x7277F7849966426d345D8F6B9AFD1d3d89183083': ['scroll'],
         },
         'failed': {
             '0xc37b40ABdB939635068d3c5f13E7faF686F03B65': [
                 'polygon_pos',
                 'arbitrum_one',
                 'base',
                 'gnosis',
+                'scroll',
             ],
         },
         'existed': {'0x7277F7849966426d345D8F6B9AFD1d3d89183083': ['gnosis']},
         'no_activity': {
             '0x106B62Fdd27B748CF2Da3BacAB91a2CaBaeE6dCa': ['all'],
-            '0x7277F7849966426d345D8F6B9AFD1d3d89183083': [
```

Comment:
can you maybe use another address so that we can check the no_activity key with multiple results and not only the `'all'` result in a test?

Reply:
addressed in https://github.com/rotki/rotki/pull/7584/commits/a72c9e4587bf541c44eb6f57f50aed3250a07ae2